watson/cli.py

```diff
@@ -997,8 +1020,12 @@ def frames(watson):
               help="Date and time of start of tracked activity")
 @click.option('-t', '--to', required=True, type=Date,
               help="Date and time of end of tracked activity")
+@click.option('--confirm-new-project', is_flag=True, default=False,
+              help="Confirm addition of new project.")
+@click.option('--confirm-new-tag', is_flag=True, default=False,
```

Comment:
For those two new options, can we:

1. default to `True` (and thus update the long option name)
2. add a short option?

Reply:
1) If I recall what I originally did (sorry, this feature has been dormant for some time) if user wants these as default they can set it in the config.
2) Watson command line is starting to run out short option names, at least in such a way that they are logically comparable to the long one.

Reply:
@jmaupetit  - you would prefer these confirmation steps become the default configuration/behaviour of watson? Doesn't that seem like a major version update kind of change? (has the potential to break scripting that people have built around watson that isn't set up to handle a user input step)

Reply:
@medwig you are right, this is a BC and we should avoid it. I should have thought more before answering this morning. So. Your implementation, is the desired behavior. To me it should be the default, so, if it's configurable as @braincow says, it sounds perfect to me.

Even if we are running out of relevant short option names, I still would prefer having one that is not that obvious than none.